js/artFeature.js

```diff
@@ -39,7 +39,7 @@ submitArtForm.addEventListener('submit', (e) => {
   };
 
   if (artTitle && htmlCode) {
-    fetch('https://api.realdevsquad.com/arts/user/add', {
+    fetch('https://api-realdevsquad.com/arts/user/add', {
```

Comment:
Is this a new route ?
just wanted to know :)

Reply:
fixed this